src/controllers/user.controller.js

```diff
@@ -0,0 +1,9 @@
+const userService = require('../services/user.service');
+
+const getAll = async (req, res) => {
+  const users = await userService.getAll();
```

Comment:
Consider adding error handling for the `userService.getAll()` call to manage potential errors and prevent unhandled promise rejections.